components/calendar/calendar-event.js

```diff
@@ -0,0 +1,43 @@
+import { LitElement } from 'lit';
+
+/**
+ * A component can be used to display events in d2l-calendar.
+ */
+class CalendarEvent extends LitElement {
+
+	static get properties() {
+		return {
+			/**
+			 * End date
+			 * @type {string}
+			 */
+			endDate: { attribute: 'end-date', type: String },
```

Comment:
Wondering if we should be using `endValue` and `startValue` instead, since "value" seems to have been used elsewhere for date/time components.

Reply:
Yeah I think aligning with `input-date-time-range` on `*-value` makes sense here.